GVRf/Framework/framework/src/main/java/org/gearvrf/GVRDepthShader.java

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gearvrf;
+
+import java.util.HashMap;
+
+import org.gearvrf.utility.TextFile;
+
+import android.content.Context;
+
+import org.gearvrf.R;
+
+   /**
+    * Manages a set of variants on vertex and fragment shaders from the same source
+    * code.
+    */
+   public class GVRDepthShader extends org.gearvrf.GVRShaderTemplate
```

Comment:
Why org.gearvrf.GVRShaderTemplate instead of GVRShaderTemplate? It may be better to make it consistent with other files.